server/auth/store.go

```diff
@@ -43,6 +43,8 @@ var (
 	authEnabled   = []byte{1}
 	authDisabled  = []byte{0}
 
+	rootPerm = authpb.Permission{PermType: authpb.READWRITE, Key: []byte("*"), RangeEnd: []byte{0}}
```

Comment:
define []byte("*") as authAllKeys? So that you can reuse it in the test at line 309.

Reply:
fine.